src/test/java/uk/gov/pay/api/utils/mocks/ConnectorMockClient.java

```diff
@@ -130,46 +127,73 @@ private String buildChargeEventsResponse(String chargeId, List<Map<String, Strin
     String nextUrlPost() {
         return "http://frontend_card/charge/";
     }
-    
-    String captureUrlPost() {
-        return "http:///";
-    }
 
     private String chargeEventsLocation(String accountId, String chargeId) {
         return baseUrl + format(CONNECTOR_MOCK_CHARGE_EVENTS_PATH, accountId, chargeId);
     }
+    
+    public void respondOk_whenCreateCharge(String gatewayAccountId, CreateChargeRequestParams requestParams) {
+
+        var responseFromConnector = aCreateChargeResponseFromConnector()
+                .withAmount(requestParams.getAmount())
+                .withChargeId("chargeId")
+                .withState(new PaymentState("created", false, null, null))
+                .withReturnUrl(requestParams.getReturnUrl())
+                .withDescription(requestParams.getDescription())
+                .withReference(requestParams.getReference())
+                .withEmail("email@email.com")
+                .withPaymentProvider("Sandbox")
+                .withGatewayTransactionId("gatewayTransactionId")
+                .withCreatedDate(SDF.format(new Date()))
+                .withLanguage(SupportedLanguage.ENGLISH)
+                .withDelayedCapture(false)
+                .withCardDetails(new CardDetails("1234", "123456", "Mr. Payment", "12/19", null, "Mastercard"))
+                .withLink(validGetLink(chargeLocation(gatewayAccountId, "chargeId"), "self"))
+                .withLink(validGetLink(nextUrl("chargeTokenId"), "next_url"))
+                .withLink(validPostLink(nextUrlPost(), "next_url_post", "application/x-www-form-urlencoded", getChargeIdTokenMap("chargeTokenId")));
+        
+        if (!requestParams.getMetadata().isEmpty()) responseFromConnector.withMetadata(requestParams.getMetadata());
```

Comment:
I think its much clearer to use `{}` especially if viewing things on the github diff...